DirectEmployers/MyJobs
# [PD-2192] Implement new tags control

This replaces the original Tags prototype with the new `TagAnd` component.

There is also a `SelectByNameOrTag` component present in this PR but I'm not using it at this time. I decided not to delete it as it is styled and more or less functions. We will need it in the future when we allow "foreign" tag filtering. (i.e. filter contacts by partner tags)

## Test 

1. Set up a report .
2. Play with the tags interface. Also play with the two-pane selects.
3. Run some reports and verify that the results are sane.

## Review

This is a from scratch rewrite of the tags interface based on design by @misterjonscott .

There are no back end changes except for tweaking a template to remove a stylesheet link.
